docs/platform/custom-domains.mdx

```diff
@@ -34,14 +34,13 @@ fqdn = ['hasura.custom-domain.com']
 [[auth.resources.networking.ingresses]]
 fqdn = ['auth.custom-domain.com']
 
-[[postgres.resources.networking.ingresses]]
-fqdn = ['postgres.custom-domain.com']
-
 [[functions.resources.networking.ingresses]]
 fqdn = ['functions.custom-domain.com']
 ```
 
-For Run services, typically in nhost-service.toml specific to the service:
+For PostgreSQL there is nothing to configure on Nhost's side so as long as you have a CNAME properly configured it should work.
+
+For Run services, typically in an `nhost-service.toml` specific to the service:
 
```

Comment:
Also maybe, in the doc, a tab or link redirecting to how to setup custom domains in the the sdk would be useful